python/python/Ice/ObjectAdapter.py

```diff
@@ -73,16 +73,12 @@ def waitForHold(self) -> None:
         """
         Wait until the object adapter holds requests.
 
-        Calling `hold` initiates holding of requests, and `waitForHold` only returns when holding of requests has been completed.
-        """
-        #
-        # TODO should be part of the documented behavior above. Should we add a timeout parameter with a default value?
```

Comment:
This TODO was bogus, the comment is about the IcePy implementation. We call it in a loop to avoid blocking the main thread.